ducktape/tests/reporter.py

```diff
@@ -142,75 +152,77 @@ def report(self):
 
 
 class HTMLSummaryReporter(SummaryReporter):
+    def __init__(self, session_id, result_store, results_dir):
+        super(HTMLSummaryReporter, self).__init__(session_id, result_store)
+        self.results_dir = results_dir
 
-    def format_test_name(self, result):
-        lines = ["Module: " + result.test_context.module_name,
-                 "Class:  " + result.test_context.cls_name,
-                 "Method: " + result.test_context.function_name]
+    def format_test_name(self, datum):
+        lines = ["Module: " + datum["module_name"],
```

Comment:
Haven't seen where this all gets included in the test "datum" yet, but it seems like there are quite a few things at the top level. Should we group all of this in a sub-object with a name that is very unlikely to conflict with anything reported by the tests into the result object?